### PR TITLE
NO-ISSUE: ART bash parser is failing on [[...]] syntax

### PR DESCRIPTION
**Pull request by jupierce**
Replace with sh compatible syntax.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -58,7 +58,7 @@ RUN dnf clean all
 COPY manifests /manifests
 
 ARG TAGS=''
-RUN if [[ "${TAGS}" = *scos* ]]; then sed -i 's/rhel-coreos/stream-coreos/g' /manifests/*; fi
+RUN if echo "${TAGS:-}" | grep scos > /dev/null; then sed -i 's/rhel-coreos/stream-coreos/g' /manifests/*; fi
```
Comment (aleskandro) [inline Dockerfile:61]:
```suggestion
RUN if echo "${TAGS:-}" | grep -q scos > /dev/null 2>&1; then sed -i 's/rhel-coreos/stream-coreos/g' /manifests/*; fi
```

**Comment by jupierce** [comment]
/override ci/prow/e2e-aws
/override ci/prow/e2e-upgrade
change is a no-op and fixes ART parsing.

**Pull request merged**